package.json

```diff
@@ -319,10 +320,15 @@
                     "group": "images_2_general@2"
                 },
                 {
-                    "command": "vscode-docker.images.tag",
+                    "command": "vscode-docker.images.pull",
                     "when": "view == dockerImages && viewItem == image",
                     "group": "images_2_general@3"
```

Comment:
I couldn't decide whether this should go next to Push, or down in the "destructive" group by Remove.